Homework-Functions/src/sciit/homework/functions/ex5/DuplicateRemoval.java

```diff
@@ -0,0 +1,60 @@
+package sciit.homework.functions.ex5;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+import static sciit.homework.functions.ex4.AverageCalculation.insertNrInArray;
+
+public class DuplicateRemoval {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("How long is the array");
+        int arrLength = sc.nextInt();
+
+        int[] arrNumbers = insertNrInArray(arrLength);
+        System.out.println(Arrays.toString(arrNumbers));
+
+        Arrays.sort(arrNumbers);
+
+
+        int[] finalArray = removeDuplicate(arrNumbers);
+        System.out.println(Arrays.toString(finalArray));
+
+    }
+
+    public static int[] removeDuplicate(int[] arrayNumbers) {
```

Comment:
Daca algoritmul tau de eliminare a duplicatelor se bazeaza pe faptul ca array-ul trebuie sa fie sortat inainte, atunci sortarea ar trebui sa se faca in interiorul acestei functii. Astfel functia poate fi apelata si cu siruri nesortate.